src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.rows];
+
+rows.forEach((row) => {
+  const cellContent = row.cells[1].innerHTML;
+  const newCells = row.insertCell(row.cells.length - 1);
+  const th = `<th>${cellContent}</th>`;
+  const td = `<td>${cellContent}</td>`;
+  const nameTableSection = row.parentNode.tagName;
+
+  if (nameTableSection === 'THEAD' || nameTableSection === 'TFOOT') {
+    newCells.outerHTML = th;
+  } else {
+    newCells.outerHTML = td;
+  }
```

Comment:
This is a code quality issue. Your code is not very DRY (Don't Repeat Yourself) here. You're creating `th` and `td` elements with the same content. Consider creating the cell content once and then decide whether to wrap it in `th` or `td` tags.